GitTfs/Core/GitRepository.cs

```diff
@@ -697,7 +697,7 @@ public bool Checkout(string commitish)
         {
             try
             {
-                _repository.Checkout(commitish);
+                LibGit2Sharp.Commands.Checkout(_repository, commitish);
```

Comment:
hmm, looks like they did away with the extension method, or was this a style preference?